gym_sokoban/envs/sokoban_env.py

```diff
@@ -119,7 +123,7 @@ def _push(self, action):
         :param action:
         :return: Boolean, indicating a change of the room's state
         """
-        change = CHANGE_COORDINATES[(action - 1) % 4]
+        change = CHANGE_COORDINATES[(action) % 4]
```

Comment:
Actions are all smaller than 4
```suggestion
        change = CHANGE_COORDINATES[action]
```

and let it fail with IndexError if it's the wrong number